Makefile

```diff
@@ -0,0 +1,11 @@
+check-code:
```

Comment:
Why can't these commands be incorporated into the package.json file?

Reply:
they already are. the idea is to have the same command in other repo, including backend, so that one don't have to remember if `make` or `yarn` has to be used